projects/meta_cl/experiments/dendrites.py

```diff
@@ -105,17 +109,21 @@ def run_experiment(config):
 
 
 # An updated version of the dendrites model.
-# |--------------------------------------------------------------|
-# |   Num Classes | Meta-test test   | Meta-test train   |    LR |
-# |--------------:|:-----------------|:------------------|------:|
-# |            10 | 0.00 ± 0.00      | 0.00 ± 0.00       | 0.001 |
-# |--------------------------------------------------------------|
+# |---------------------------------------------------------------|
```

Comment:
We quickly discussed about this, but it is significantly faster if we don't gridsearch LR during test time. We can still do that for final results, but I don't clearly see the benefits of doing that every run during experimentation.

It can be argued that there might be better LRs for each method/architecture, but that can be said for any hyperparameter. It is hard to put the finger on what is a good trade-off during experimentation. But I do feel that meta-testing as it is takes a long time and its slowness is hindering progress (it did for me at least, I turned off when running experiments).

A second argument, but more opinionated, is that I don't believe it fits the concept of continual learning to do hyperparameter search during test time, specially given the test data distribution is similar to the one seen during meta-training. There is no application scenario I can imagine that fits this odd paradigm, this is purely an artifact of ANML/OML starting from a copy of the earlier MAML implementation. The optimal LR should be picked up as part of the meta learning process.

Reply:
All good points. I can include an option to skip it. I'd like to keep it to enable fair comparisons to OML and ANML.